src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -199,10 +265,13 @@ ChipError::StorageType pychip_DeviceController_NewDeviceController(chip::Control
     const chip::Credentials::AttestationTrustStore * testingRootStore = chip::Credentials::GetTestAttestationTrustStore();
     SetDeviceAttestationVerifier(GetDefaultDACVerifier(testingRootStore));
 
-    CHIP_ERROR err = sOperationalCredentialsIssuer.Initialize(sStorageDelegate);
-    VerifyOrReturnError(err == CHIP_NO_ERROR, err.AsInteger());
+    chip::Controller::ExampleOperationalCredentialsIssuer * operationalCredentialsIssuer =
```

Comment:
I did not see any place freeing this pointer

Possible solution is to create a new class inherits from chip::Controller::DeviceCommissioner with this member and make this function a wrapper of chip::Controller::Python::DeviceCommissioner() (or Init, or something else)

Maybe a TODO here is OK?

Reply:
Good catch. I'll free it in `pychip_DeviceController_DeleteDeviceController`, which is the symmetric equivalent here.